adoorback/notification/serializers.py

```diff
@@ -30,8 +30,14 @@ def get_is_friend_request(self, obj):
 
     def get_recent_actors(self, obj):
         from account.serializers import UserMinimalSerializer
-        recent_actors = obj.actors.all()[:3]
-        return UserMinimalSerializer(recent_actors, many=True).data
+        if obj.target and obj.target._meta.model_name == 'ping':
+            actor = obj.actors.first()
+            count = obj.actors.count()
+            actors = [actor] * count 
+            return UserMinimalSerializer(actors, many=True).data
```

Comment:
`recent_actors` field is used on the frontend side to display profile images in the notification.
It's a bit awkward to display 3 same images, so I think it's better to return only one actor.
<img width="493" alt="스크린샷 2025-03-16 오후 2 54 20" src="https://github.com/user-attachments/assets/85358a00-2ff1-4e4a-adda-52a6986db953" />

```suggestion
        if obj.target and obj.target._meta.model_name == 'ping':
            return UserMinimalSerializer([obj.actors.first()], many=True).data
```